pkg/model/components/etcdmanager/model.go

```diff
@@ -196,6 +198,10 @@ spec:
       path: /
       type: Directory
     name: rootfs
+  - hostPath:
+      path: /etc/ssl
+      type: Directory
+    name: ca-certificates
```

Comment:
I'm a little torn about this.  On the one hand, it's great that we pick up system ca-certificates so if they update we don't have to do anything (although in practice these don't change fast, but I guess if someone wanted to install their _own_ in-house ca-certificate this now work).  On the other hand, using hostPath to pick up ca-certificates isn't the general pattern for other pods, and I don't know if we're going to have new problems.

Given it is etcd-manager, which is really a system component and in another universe could run as a systemd service, I say we run with it!